src/main/steps/applicant1/finalising-your-application/template.njk

```diff
@@ -9,15 +9,94 @@
     <p class="govuk-body">{{ readMore.line3 }}</p>
 {% endset %}
 
+
+{% set readMoreAboutSettlingFinancesContentJoint %}
+  <p class="govuk-body">{{ readMoreAboutSettlingFinances.line1 }}</p>
+  <p class="govuk-body">{{ readMoreAboutSettlingFinances.line2 }}</p>
+  <p class="govuk-body">{{ readMoreAboutSettlingFinances.line3.part1 }}
+    <a href="{{ readMoreAboutSettlingFinances.line3.link }}">
+      {{ readMoreAboutSettlingFinances.line3.part2 }}
+    </a>
+    {{ readMoreAboutSettlingFinances.line3.part3 }}
+  </p>
+{% endset %}
+
+{% set readMoreContentChangeToSole %}
+  <p class="govuk-body">{{ readMoreChangeToSole.line1 }}</p>
+  {% if isAwaitingJointFinalOrderState  %}
+    <p class="govuk-body">{{ readMoreChangeToSole.line2 }}</p>
+  {% endif %}
+  <ol class="govuk-list govuk-list--number">
+    <li>
+      <p class="govuk-body"><a href="{{ links.applicationForFinalOrder }}">
+          {{ readMoreChangeToSole.orderedList1.part1 }}</a>
```

Comment:
maybe should also be called linkText, to be consistent with the rest of the content

Reply:
Thanks @benouaer I made those changes.